### DISCUSSION TITLE
EntryNav ID4 "Connection assistant"

### DISCUSSION THREAD

**Discussion by matthewblabey** [BimmerGestalt/AAIdrive#387] [Q&A]
I'm trying to get your app to work with the EntryNav ID4. Do you know if this version of ID will work with the app? I bought the basic connected drive services yesterday so I could enable the "Connection assistant option" as advised in the README but I just checked after 24 hours of buying connected drive and I don't have the "Connection Assistant" in the car.

I managed to install the older app but when I try to sign in to my connected drive account, it says "An authentication error occurred. Please try again." Seems to me you cant sign into anymore since MyBMW exists?

This app will certainly be a lot cheaper option for me than just retrofitting an entire new iDrive. Hope you can help :)

**Comment by hufman** [comment]
Yes I believe ID4 should still work. I'm not sure how long after buying the subscription it takes before the car notices, as #384 is encountering the same thing. It may be that this generation of car has shut down its subscription service, and that it must have the 6NR option locally coded from now on? I would like confirmation from others on the specific subscription that they needed use too, I'm not sure which tier is the specific one that enables the apps.

I believe they disabled BMW Connected app authentication to attempt to shut down the app and switch to MyBMW. Connected version 6.4 does still work for local car connections before logging in, but the car must have the Connection Assistant option first. BMW Connected Classic has also been used successfully, sometimes even better for ID4 cars.

Hopefully it can be figured out soon!

**Reply by matthewblabey** [reply]
Please ignore everything i said, about an hour a go the connection assistant option has appeared and the app worked instantly.
